Documentation/bpf/progtypes.rst

```diff
@@ -338,49 +338,88 @@ corresponding network driver name of an interface can be determined as follows:
-|                   | ice        | >= 5.5      |
-+-------------------+------------+-------------+
-| Marvell           | mvneta     | >= 5.5      |
-+-------------------+------------+-------------+
-| Mellanox          | mlx4       | >= 4.8      |
-|                   +------------+-------------+
-|                   | mlx5       | >= 4.9      |
-+-------------------+------------+-------------+
-| Microsoft         | hv_netvsc  | >= 5.6      |
-+-------------------+------------+-------------+
-| Netronome         | nfp        | >= 4.10     |
-+-------------------+------------+-------------+
-| Others            | virtio_net | >= 4.10     |
-|                   +------------+-------------+
-|                   | tun/tap    | >= 4.14     |
-|                   +------------+-------------+
-|                   | bond       | >= 5.15     |
-+-------------------+------------+-------------+
-| Qlogic            | qede       | >= 4.10     |
-+-------------------+------------+-------------+
-| Socionext         | netsec     | >= 5.3      |
-+-------------------+------------+-------------+
-| Solarflare        | sfc        | >= 5.5      |
-+-------------------+------------+-------------+
-| Texas Instruments | cpsw       | >= 5.3      |
-+-------------------+------------+-------------+
++------------------------+-------------------------+-------------+
+| Vendor                 | Driver                  | XDP Support |
++========================+=========================+=============+
+| Amazon                 | ena                     | >= 5.6      |
++------------------------+-------------------------+-------------+
+| Aquantia               | atlantic                | >= 5.13     |
++------------------------+-------------------------+-------------+
+| Broadcom               | bnxt_en                 | >= 4.11     |
++------------------------+-------------------------+-------------+
+| Cavium                 | thunderx                | >= 4.12     |
++------------------------+-------------------------+-------------+
+| Engleder               | tsne                    | >= 6.3      |
+|                        | (TSN Express Path)      |             |
++------------------------+-------------------------+-------------+
+| Freescale              | dpaa                    | >= 5.0      |
+|                        +-------------------------+-------------+
+|                        | dpaa2                   | >= 5.11     |
+|                        +-------------------------+-------------+
+|                        | enetc                   | >= 5.13     |
+|                        +-------------------------+-------------+
+|                        | fec_enet                | >= 6.2      |
++------------------------+-------------------------+-------------+
+| Fungible               | fun                     | >= 5.18     |
++------------------------+-------------------------+-------------+
+| Google                 | gve                     | >= 5.19     |
```

Comment:
If I am not mistaken support was added in v6.4 [see](https://github.com/torvalds/linux/commit/75eaae158b1b7d8d5bde2bafc0bcf778423071d3)